src/utils.js

```diff
@@ -2,6 +2,11 @@ import defaults from './options'
 
 function format (input, opt = defaults) {
```

Comment:
@codymoorhouse 
Input var can be a string. If decimal is ',' then input can be like '10,000'